guides/source/development_dependencies_install.md

```diff
@@ -43,215 +43,113 @@ $ git clone https://github.com/rails/rails.git
 
-```bash
-$ gem install bundler
-$ gem update bundler
-```
+NOTE: Redis' documentation discourage installations with package managers as those are usually outdated. Installing from source and bringing the server up is straight forward and well documented on [Redis' documentation](https://redis.io/download#installation).
 
-and run:
+NOTE: Active Record tests _must_ pass for at least MySQL, PostgreSQL, and SQLite3. Subtle differences between the various adapters have been behind the rejection of many patches that looked OK when tested only against single adapter.
 
-```bash
-$ bundle install --without db
-```
+Below you can find instructions on how to install all of the additional tools for different OSes.
 
-This command will install all dependencies except the MySQL and PostgreSQL Ruby drivers. We will come back to these soon.
+#### macOS
 
-NOTE: If you would like to run the tests that use memcached, you need to ensure that you have it installed and running.
+On macOS you can use [Homebrew](https://brew.sh/) to install all of the additional tools.
 
-You can use [Homebrew](https://brew.sh/) to install memcached on macOS:
+To install all run:
 
 ```bash
-$ brew install memcached
+$ brew bundle
 ```
 
-On Ubuntu you can install it with apt-get:
+You'll also need to start each of the installed services. To list all available services run:
 
 ```bash
-$ sudo apt-get install memcached
+$ brew services list
 ```
 
-Or use yum on Fedora or CentOS:
+You can then start each of the services one by one like this:
 
 ```bash
-$ sudo yum install memcached
+$ brew services start mysql
 ```
 
-If you are running on Arch Linux:
+Replace `mysql` with the name of the service you want to start.
 
-```bash
-$ sudo pacman -S memcached
-```
+#### Ubuntu
 
-For FreeBSD users, you're done with:
+To install all run:
 
 ```bash
-# pkg install memcached
-```
-
-Alternatively, you can compile the `databases/memcached` port.
-
-With the dependencies now installed, you can run the test suite with:
-
-```bash
-$ bundle exec rake test
-```
-
-You can also run tests for a specific component, like Action Pack, by going into its directory and executing the same command:
-
-```bash
-$ cd actionpack
-$ bundle exec rake test
+$ sudo apt-get update
+$ sudo apt-get install sqlite3 libsqlite3-dev mysql-server libmysqlclient-dev postgresql postgresql-client postgresql-contrib libpq-dev redis-server memcached imagemagick ffmpeg mupdf mupdf-tools
+# install yarn
+$ curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+$ echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+$ sudo apt-get install yarn
 ```
 
-If you want to run the tests located in a specific directory use the `TEST_DIR` environment variable. For example, this will run the tests in the `railties/test/generators` directory only:
-
-```bash
-$ cd railties
-$ TEST_DIR=generators bundle exec rake test
```

Comment:
Since you've removed all the testing instructions because they are all explained in the "Contributing" guide, it looks like this information is not present elsewhere. Maybe you can add a "Running all tests inside a directory" section to the contributing guide near the ["Running a single test"](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#running-a-single-test) section. What do you think ?